tests/generic_config_updater/test_cacl.py

```diff
@@ -42,6 +42,18 @@ def get_iptable_rules(duthost):
     return rules_chain
 
 
+@pytest.fixture(scope="module", autouse=True)
+def disable_port_toggle(duthosts, tbinfo):
```

Comment:
Since it's same piece of code, will it be worth to move it to dualtor common or dualtor utils, and call the fixture in tests?

It's up to you, not mandatory.

Reply:
@zjswhhh This new fixture is module fixture, which only works within script. If I move it to dualtor utils, it can't be used in specific script as module level and autouse.
All fixtures in dualtor_tor_common.py is session level, and I don't want to set max mode to manual for all test cases.